.github/workflows/test.yml

```diff
@@ -49,20 +49,22 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            wasmer-feature: wasmer_sys
+            target: wasmer_sys
           - os: macos-latest
-            wasmer-feature: wasmer_sys
+            target: wasmer_sys
           - os: macos-13
-            wasmer-feature: wasmer_sys
+            target: wasmer_sys
           - os: windows-latest
-            wasmer-feature: wasmer_sys
+            target: wasmer_sys
+          - os: ubuntu-latest
+            target: unstable
```

Comment:
We're starting to get a few of these. I guess we'll just have to keep an eye on the balance between CI taking time waiting for runner instances and taking too long on single runs.

Reply:
Yeah I agree

At least for these unstable features, I hope they can all go under one test target here